src/components/hooks/useWS.ts

```diff
@@ -3,7 +3,7 @@ import { useDerivApi } from './use-deriv-api'
 import type { TSocketEndpointNames, TSocketRequestProps, TSocketResponseData } from 'types/types'
 
 const useWS = <T extends TSocketEndpointNames>(name: T) => {
-    const [is_loading, setIsLoading] = useState(false)
+    const [is_region_loading, setIsLoading] = useState(false)
```

Comment:
@mohammad-rework Same here, I guess you did replace all which affected these files as well 🤔